stable/rabbitmq/Chart.yaml

```
@@ -1,5 +1,5 @@
 name: rabbitmq
-version: 4.1.0
+version: 4.1.1
```

you are adding a new feature so following semver versioning you should update the minor version.

```suggestion
version: 4.2.0
```